### ISSUE TITLE
Feature request: A way to export data per-dwarf or other entity in a spreadsheet/csv or some such format

### ISSUE THREAD

**Issue by kholat** [Dwarf-Therapist/Dwarf-Therapist#258]
So im trying to do some data analysis outside of the game because obviously writing things down 1-by-1 is a PITA, and I was looking for a tool that kinda exports all the data in a spreadsheet (like the screen that has preferences for instance) but I only managed to come across this reddit thread https://www.reddit.com/r/dwarffortress/comments/mknon/anyone_know_of_a_dwarf_relationship_utility/ and the following comment is pretty much my question but 10 years later: 
_"that's what I've been doing x_x;

it's horrible, going through the dwarfs one. profile. at. a. time. --driving me stir-crazy at the task! I'm only about 18% through the population, and that's a GENEROUS estimate.. I really wish I could dump all that into a spreadsheet directly from the game data. Right now that is the only method to view it. Interface design is not the dev's strong suit... -_-;

AT LEAST I found I can get straight to a "UNIT LIST" by hitting u at the main screen... *shudder* no more wading through that bloody task list or Z-Health menu..."_

I do know that there is an export feature for per-view, but its not what i want. For instance if I want to see just each dwarves name and age, it doesnt work that way.

Basically im wondering if theres a way to output some wanted data to analyze it with 3rd party software like excel, wolfram mathematica etc.

Thanks in advance

**Comment by cvuchener** [comment]
If the current export to csv feature is not enough for you, you should use a DFHack script (as it was already suggested 10 years ago). I can't add very specific features that will only be used by one person.

**Comment by kholat** [comment]
Thats fine thanks, doesnt hurt to ask
